src/dune_rules/dune_package.ml

```diff
@@ -86,7 +86,7 @@ module Lib = struct
        ; field_o "default_implementation" (no_loc Lib_name.encode)
            default_implementation
        ; field_o "main_module_name" Module_name.encode main_module_name
-       ; field_l "modes" sexp (Mode.Dict.Set.encode modes)
+       ; field_l "modes" sexp (Lib_mode.Dict.Set.encode modes)
```

Comment:
I think we should use the old modes here for all the decoders. We don't support installing melange artifacts yet.

Reply:
Removed in https://github.com/jchavarri/dune/pull/4/commits/9bbdd97b08870aa1048ab1afbc26dddb9b55701b.